e2e/testnet_wallet.test.js

```diff
@@ -0,0 +1,155 @@
+
+  // change to testnet mode
+  await waitFor(element(by.id('header.drawmenu'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id('header.drawmenu')).tap();
+  await waitFor(element(by.id('menu.settings'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id('menu.settings')).tap();
+  await element(by.id('settings.scrollView')).scroll(1000, 'down');
+  // waiting for the custom server radio
+  // choose the custom server 
+  await waitFor(element(by.id('settings.customServer'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id('settings.customServer')).tap();
+
+  // waiting for the custom server field
+  await waitFor(element(by.id('settings.customServerField'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id("settings.customServerField")).replaceText('https://testnet.lightwalletd.com');
+  await waitFor(element(by.id('chaintype.testnet'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id("chaintype.testnet")).tap();
+
+  // save the new testnet server
+  await waitFor(element(by.id('settings.button.save'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id('settings.button.save')).tap();
+
+  await waitFor(element(by.id('seed.button.OK'))).toBeVisible().withTimeout(ui_timeout);
+  // three taps to be really sure...
+  await element(by.id('seed.button.OK')).tap();
+  await element(by.id('seed.button.OK')).tap();
+  await element(by.id('seed.button.OK')).tap();
+
+  log.info('onto loadingapp screen')
+}
+
+async function load(seed,birthday) {
+  await waitFor(element(by.id('loadingapp.restorewalletseed'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id("loadingapp.restorewalletseed")).tap();
+  await waitFor(element(by.id('seed.seedinput'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id("seed.seedinput")).replaceText(seed);
+  await waitFor(element(by.id('seed.birthdayinput'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id("seed.birthdayinput")).replaceText(birthday);
+  await waitFor(element(by.id('seed.button.OK'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id('seed.button.OK')).tap();
+}
+
+async function sync(background_duration) {
+  await waitFor(element(by.id('header.drawmenu'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id('header.drawmenu')).tap();
+  await waitFor(element(by.id('menu.syncreport'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id('menu.syncreport')).tap();
+
+  // waiting for starting the sync process
+  await waitFor(element(by.id('syncreport.currentbatch'))).toBeVisible().withTimeout(ui_timeout);
+
+  // put the App in background
+  await device.sendToHome();
+  // put the App to sleep because we need some progress in background sync
+  await sleep(background_duration);
+  // put the App in foregroung again
+  await device.launchApp({ newInstance: false });
+
+  await waitFor(element(by.text("Finished"))).toBeVisible().withTimeout(sync_timeout)
```

Comment:
Maybe 10 seconds for `background_duration` is not enough, just an idea, try to make this bigger...